### PR TITLE
Add \orcids command to add support for multiple ORCIDs

### PR DESCRIPTION
**Pull request by robertfromont**
`\orcids` takes a list of ORCID urls, delimited by semicolons, following the apparent pattern in published articles, e.g.
```
\orcids{https://orcid.org/0000-1234-5678-9999;https://orcid.org/0000-5678-1234-9999}
```

### CODE REVIEW

File: ldr-article.cls [hunk 1]
```diff
@@ -88,8 +91,18 @@
...
+% for a single ORCID
 \newcommand{\orcid}[1]{\titlepagefontsize\textbf{ORCID ID: }\url{#1} \\[\baselineskip]}
 
+% for a multiple ORCIDs delimited by semicolon ;
+\newcommand{\orcids}[1]{\titlepagefontsize\textbf{ORCID IDs: }\orcidlist{#1} \\[\baselineskip]}
+\NewDocumentCommand{\orcidlist}{ >{\SplitList{;}} m }{%
+  \ProcessList{#1}{\orciditem}%
+}
+\NewDocumentCommand{\orciditem}{m}{%
+  \url{#1}; % a space follows
```
Comment (mitjanikolaus) [inline ldr-article.cls:103]: would there be an easy way to not having a trailing semicolon after the last element?

Comment (robertfromont) [reply ldr-article.cls:103]: I did try to figure that out, but the xparse package documentation didn't appear to offer a way to traverse a list with any indication of whether the current item is the last one or not.

LaTeX isn't my forte, though, so it's possible some way exists that I couldn't figure out!

Comment (mitjanikolaus) [reply ldr-article.cls:103]: mine neither, but it looks like I found a solution: https://github.com/mitjanikolaus/ldr-template/commit/8f0293858f7d4fdee2d97b094418337efd10be27 :)

Comment (robertfromont) [reply ldr-article.cls:103]: Clever solution!

**Pull request merged**